src/components/DataProvider/index.js

```diff
@@ -739,6 +745,8 @@ DataProvider.propTypes = {
   // loaderConfig => void
   // called whenever data is fetched by the loader
   onFetchData: PropTypes.func,
+  // Callback when fetching data from backend returns an error
```

Comment:
Can you document what this signature looks like? Also, it's more of when the loader throws an exception -- we don't know that it's loading from a backend